OrbitCore/Capture.cpp

```diff
@@ -51,6 +51,7 @@ std::string Capture::GProcessName;
 std::unordered_map<int32_t, std::string> Capture::GThreadNames;
 std::unordered_map<uint64_t, LinuxAddressInfo> Capture::GAddressInfos;
 std::unordered_map<uint64_t, std::string> Capture::GAddressToFunctionName;
+std::unordered_map<uint64_t, std::string> Capture::GAddressToModuleName;
```

Comment:
Can these be moved to samplingprofilier? maybe not in this PR.

Reply:
Added it to my TODO list